src/utils/standard_paths.cpp

```diff
@@ -40,3 +42,22 @@ QString mp::StandardPaths::writableLocation(StandardLocation type) const
 {
     return QStandardPaths::writableLocation(type);
 }
+
+std::filesystem::path mp::get_default_daemon_path()
+{
+    using Path = std::filesystem::path;
+    const auto base_dir =
+        utils::in_multipass_snap()
+            ? Path{utils::snap_common_dir().toStdString()} / "data"
+            : Path{MP_STDPATHS.writableLocation(StandardPaths::GenericDataLocation)
```

Comment:
This would return ~/.local/share, and ~ would resolve to `/root` and `/home/<user` depending on the authorization level (root vs regular user).

Reply:
The client can connect to the gRPC server with `sudo`:

sudo ./build/bin/multipass list
[sudo] password for mkg: 
list failed: The user is not authenticated with the Multipass service.

Please authenticate before proceeding (e.g. via 'multipass authenticate'). Note that you first need an authenticated user to set and provide you with a trusted passphrase (e.g. via 'multipass set local.passphrase').